components/checkbox/stories/checkbox.stories.js

```diff
@@ -115,19 +121,85 @@ const CheckboxGroup = (args) => html`
 			...args,
 			isIndeterminate: true,
 		})}
-			${Template({
+		${Template({
 			...args,
 			isDisabled: true,
 		})}
 		${Template({
 			...args,
-			label: "Checkbox with wrapping label text",
-			customStyles: { "max-inline-size": "100px" },
+			label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+      customStyles: { "max-inline-size": "200px" },
 		})}
 	</div>
 `;
 
+const AllVariantsCheckboxGroup = (args) => {
```

Comment:
Let's do 2 things with this -

1. move the markup function here to the template.js file as an export.
2. add the context passthrough (i.e. `(args, context) =>` and `Template(args, context)`)

Reply:
Can do. I took care of this for the coach indicator as well since it had some markup in the stories file: 

- checkbox: https://github.com/adobe/spectrum-css/pull/2793/commits/c567092e95ce360235933e117279d10551d59358
- coachindicator: https://github.com/adobe/spectrum-css/pull/2793/commits/402da37445171bd6f52d12cdf311a8341c099e56